<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>AdvanceMonitorDlg</class>
 <widget class="QWidget" name="AdvanceMonitorDlg">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1038</width>
    <height>617</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>AdvanceMonitorDlg</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="QTreeWidget" name="trAdvanceFileMonitor">
     <property name="maximumSize">
      <size>
       <width>200</width>
       <height>16777215</height>
      </size>
     </property>
     <property name="focusPolicy">
      <enum>Qt::NoFocus</enum>
     </property>
     <property name="styleSheet">
      <string notr="true"> QWidget
 {
	border:0px solid #cccccc;
	border-right-width:1px;
	border-bottom-width:1px;
	border-left-width:1px;
	background-color:rgb(240,240,240);
 }
QTreeWidget::item {
	min-height:30px;
	background-color:transparent;
	padding-left:5px;
 }
 QTreeWidget::item:hover {
	background-color:rgba(100,100,100, 50);
	min-height:30px;
 }
  QTreeWidget::item:selected {
	background-color:#cccccc;
	color:black;
 }
 QTreeWidget::item:!has-children
 {
	background-color:transparent;
	padding-left:25px;
 }
  QTreeWidget::item:!has-children:hover
 {
	background-color:rgba(100,100,100,50);
	padding-left:25px;
 }
 QTreeWidget::item:!has-children:selected {
	background-color:#cccccc;
	color:black;
 }
 QTreeWidget::branch {
    background: transparent;
 }</string>
     </property>
     <property name="indentation">
      <number>0</number>
     </property>
     <property name="rootIsDecorated">
      <bool>false</bool>
     </property>
     <attribute name="headerVisible">
      <bool>false</bool>
     </attribute>
     <column>
      <property name="text">
       <string>1</string>
      </property>
     </column>
     <item>
      <property name="text">
       <string>USB 设备监控</string>
      </property>
      <property name="icon">
       <iconset resource="maindlg.qrc">
        <normaloff>:/image/usbplugrecord.png</normaloff>:/image/usbplugrecord.png</iconset>
      </property>
      <item>
       <property name="text">
        <string>自动获取</string>
       </property>
       <property name="icon">
        <iconset resource="maindlg.qrc">
         <normaloff>:/image/downsearchfile.png</normaloff>:/image/downsearchfile.png</iconset>
       </property>
      </item>
      <item>
       <property name="text">
        <string>插拔记录</string>
       </property>
       <property name="icon">
        <iconset resource="maindlg.qrc">
         <normaloff>:/image/usbmonitor.png</normaloff>:/image/usbmonitor.png</iconset>
       </property>
      </item>
     </item>
     <item>
      <property name="text">
       <string>文件状态监控</string>
      </property>
      <property name="icon">
       <iconset resource="maindlg.qrc">
        <normaloff>:/image/interception.png</normaloff>:/image/interception.png</iconset>
      </property>
      <item>
       <property name="text">
        <string>监控记录</string>
       </property>
       <property name="icon">
        <iconset resource="maindlg.qrc">
         <normaloff>:/image/userfileaction.png</normaloff>:/image/userfileaction.png</iconset>
       </property>
      </item>
     </item>
    </widget>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_16">
     <property name="spacing">
      <number>0</number>
     </property>
     <item>
      <widget class="QLabel" name="lbAdvanceFileMonitor">
       <property name="minimumSize">
        <size>
         <width>0</width>
         <height>35</height>
        </size>
       </property>
       <property name="styleSheet">
        <string notr="true">QLabel
{	
	font-family:&quot;Microsoft Yahei&quot;;
	background-color: #dddddd;
	padding-left:20px;
}</string>
       </property>
       <property name="text">
        <string>USB 设备监控</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QStackedWidget" name="stkAdvanceFileMonitor">
       <property name="currentIndex">
        <number>2</number>
       </property>
       <widget class="QWidget" name="page_9">
        <layout class="QVBoxLayout" name="verticalLayout_18">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tbUsbRetrieve">
           <property name="styleSheet">
            <string notr="true">
QHeaderView::section {
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #ffffff, stop:1 #eeeeee);
	border:none;
	border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #dddddd, stop:1 #ffffff);
	border-bottom: 0px solid #eeeeee;
	height:25px;
	color:#333333;
	font-family:&quot;微软雅黑&quot;;
	text-align:center left;
 }

QTableWidget QHeaderView::section:hover {
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #ddeeff, stop:1 #eef0ff);
 }

QTableWidget:item
{
	padding-left:10px;
}

QFrame#tbUsbRetrieve{
	border:1px solid #cccccc;
	border-left-width:0px;
}

 QHeaderView::down-arrow {
     image: none;
 }

 QHeaderView::up-arrow {
     image:none;
}
 QScrollBar:vertical {
     border: 0px solid grey;
     background: transparent;
     width: 10px;
 }
 QScrollBar::handle:vertical {
     background: #cccccc;;
     min-height: 100px;
 }
 QScrollBar::handle:vertical:hover {
     background: #aaaaaa;
     min-height: 100px;
 }
 QScrollBar::add-line:vertical {
	border:none;
	background:transparent;
 }
 QScrollBar::sub-line:vertical {
	border:none;
	background:transparent;
 }</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::NoFrame</enum>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>200</number>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
           <column>
            <property name="text">
             <string>文件目标路径</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>大小</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>类型</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>修改时间</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>从U盘获取时间</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_7">
        <property name="styleSheet">
         <string notr="true">QWidget#page_7
{
	border:0px solid #aaaaaa;
	border-bottom-width:1px;
}</string>
        </property>
        <layout class="QVBoxLayout" name="verticalLayout_4">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QTableWidget" name="tbUsbPlugRecord">
           <property name="styleSheet">
            <string notr="true">
QHeaderView::section {
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #ffffff, stop:1 #eeeeee);
	border:none;
	border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #dddddd, stop:1 #ffffff);
	border-bottom: 0px solid #eeeeee;
	height:25px;
	color:#333333;
	font-family:&quot;微软雅黑&quot;;
	text-align:center left;
 }

QTableWidget QHeaderView::section:hover {
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #ddeeff, stop:1 #eef0ff);
 }

QTableWidget:item
{
	padding-left:10px;
}

QFrame#tbUsbPlugRecord{
	border:1px solid #cccccc;
	border-left-width:0px;
}

 QHeaderView::down-arrow {
     image: none;
 }

 QHeaderView::up-arrow {
     image:none;
}

 QScrollBar:vertical {
     border: 0px solid grey;
     background: transparent;
     width: 10px;
 }
 QScrollBar::handle:vertical {
     background: #cccccc;;
     min-height: 100px;
 }
 QScrollBar::handle:vertical:hover {
     background: #aaaaaa;
     min-heiht: 100px;
 }
 QScrollBar::add-line:vertical {
	border:none;
	background:transparent;
 }
 QScrollBar::sub-line:vertical {
	border:none;
	background:transparent;
 }</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>200</number>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
           <column>
            <property name="text">
             <string>插入时间</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>盘符</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>USB 设备 ID</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>时长 (秒)</string>
            </property>
           </column>
          </widget>
         </item>
         <item>
          <widget class="QTableWidget" name="tbUsbDir">
           <property name="styleSheet">
            <string notr="true">
QHeaderView::section {
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #ffffff, stop:1 #eeeeee);
	border:none;
	border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #dddddd, stop:1 #ffffff);
	border-bottom: 0px solid #eeeeee;
	height:25px;
	color:#333333;
	font-family:&quot;微软雅黑&quot;;
	text-align:center left;
 }

QTableWidget QHeaderView::section:hover {
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #ddeeff, stop:1 #eef0ff);
 }

QTableWidget:item
{
	padding-left:10px;
}

QFrame#tbUsbDir{
	border:1px solid #cccccc;
	border-top-width:0px;
	border-left-width:0px;
}

 QHeaderView::down-arrow {
     image: none;
 }

 QHeaderView::up-arrow {
     image:none;
}

 QScrollBar:vertical {
     border: 0px solid grey;
     background: transparent;
     width: 10px;
 }
 QScrollBar::handle:vertical {
     background: #cccccc;;
     min-height: 100px;
 }
 QScrollBar::handle:vertical:hover {
     background: #aaaaaa;
     min-heiht: 100px;
 }
 QScrollBar::add-line:vertical {
	border:none;
	background:transparent;
 }
 QScrollBar::sub-line:vertical {
	border:none;
	background:transparent;
 }</string>
           </property>
           <property name="editTriggers">
            <set>QAbstractItemView::NoEditTriggers</set>
           </property>
           <property name="selectionMode">
            <enum>QAbstractItemView::SingleSelection</enum>
           </property>
           <property name="selectionBehavior">
            <enum>QAbstractItemView::SelectRows</enum>
           </property>
           <property name="showGrid">
            <bool>false</bool>
           </property>
           <attribute name="horizontalHeaderDefaultSectionSize">
            <number>200</number>
           </attribute>
           <attribute name="horizontalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="horizontalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="horizontalHeaderStretchLastSection">
            <bool>true</bool>
           </attribute>
           <attribute name="verticalHeaderVisible">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderDefaultSectionSize">
            <number>20</number>
           </attribute>
           <attribute name="verticalHeaderHighlightSections">
            <bool>false</bool>
           </attribute>
           <attribute name="verticalHeaderMinimumSectionSize">
            <number>20</number>
           </attribute>
           <column>
            <property name="text">
             <string>名称</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>修改时间</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>类型</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>大小</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>属性</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
       <widget class="QWidget" name="page_6">
        <layout class="QVBoxLayout" name="verticalLayout_17">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="margin">
          <number>0</number>
         </property>
         <item>
          <widget class="QFrame" name="toolbar">
           <property name="minimumSize">
            <size>
             <width>0</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame#toolbar
{
	background-color:white;
}</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::StyledPanel</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Raised</enum>
           </property>
           <layout class="QHBoxLayout" name="horizontalLayout_32">
            <property name="spacing">
             <number>10</number>
            </property>
            <property name="leftMargin">
             <number>9</number>
            </property>
            <property name="topMargin">
             <number>4</number>
            </property>
            <property name="bottomMargin">
             <number>4</number>
            </property>
            <item>
             <widget class="QToolButton" name="btnCleanMonitorRecord">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{
	border:none;
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:transparent;
}

QToolButton:hover
{
	background-color:#d5d5d5;
}

QToolButton:pressed
{
	background-color:#bbbbbb;
}

QToolButton:disabled
{
	background-color:transparent;
}</string>
              </property>
              <property name="text">
               <string>清空记录</string>
              </property>
              <property name="icon">
               <iconset resource="maindlg.qrc">
                <normaloff>:/image/destroy.png</normaloff>:/image/destroy.png</iconset>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <widget class="QToolButton" name="btnCancelMonitor">
              <property name="minimumSize">
               <size>
                <width>80</width>
                <height>25</height>
               </size>
              </property>
              <property name="maximumSize">
               <size>
                <width>80</width>
                <height>25</height>
               </size>
              </property>
              <property name="styleSheet">
               <string notr="true">QToolButton
{
	border:none;
	font-family:&quot;Microsoft Yahei&quot;;
	background-color:transparent;
}

QToolButton:hover
{
	background-color:#d5d5d5;
}

QToolButton:pressed
{
	background-color:#bbbbbb;
}

QToolButton:disabled
{
	background-color:transparent;
}</string>
              </property>
              <property name="text">
               <string>取消监视</string>
              </property>
              <property name="icon">
               <iconset resource="maindlg.qrc">
                <normaloff>:/image/delete.png</normaloff>:/image/delete.png</iconset>
              </property>
              <property name="toolButtonStyle">
               <enum>Qt::ToolButtonTextBesideIcon</enum>
              </property>
             </widget>
            </item>
            <item>
             <spacer name="horizontalSpacer_11">
              <property name="orientation">
               <enum>Qt::Horizontal</enum>
              </property>
              <property name="sizeHint" stdset="0">
               <size>
                <width>596</width>
                <height>20</height>
               </size>
              </property>
             </spacer>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <widget class="QTreeWidget" name="trFileStatus">
           <property name="styleSheet">
            <string notr="true">/* QHeaderView::section {
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(172, 197, 224, 255), stop:1 rgba(200, 230, 255, 225));
    padding-left: 4px;
	border:none;
	border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 rgba(130, 140, 190, 255), stop:1 rgba(172, 197, 255, 225));;
	height:25px;
	font-family:&quot;微软雅黑&quot;;
 }*/


QHeaderView::section {
	background-color:qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #ffffff, stop:1 #eeeeee);
	border:none;
	border-right: 1px solid qlineargradient(spread:pad, x1:0, y1:0, x2:0, y2:1, stop:0 #dddddd, stop:1 #ffffff);
	border-bottom: 0px solid #eeeeee;
	height:25px;
	color:#333333;
	font-family:&quot;微软雅黑&quot;;
	text-align:center left;
 }
TableWidget:item
{
	padding-left:10px;
}

QFrame#trFileStatus{
	border:1px solid #cccccc;
	border-left-width:0px;
	border-right-width:0px;
}

 QHeaderView::down-arrow {
     image: none;
 }

 QHeaderView::up-arrow {
     image:none;
}

 QScrollBar:vertical {
     border: 0px solid grey;
     background: transparent;
     width: 10px;
 }
 QScrollBar::handle:vertical {
     background: #cccccc;;
     min-height: 100px;
 }
 QScrollBar::handle:vertical:hover {
     background: #aaaaaa;
     min-heiht: 100px;
 }
 QScrollBar::add-line:vertical {
	border:none;
	background:transparent;
 }
 QScrollBar::sub-line:vertical {
	border:none;
	background:transparent;
 }

 QScrollBar:horizontal {
     border: 0px solid grey;
     background: transparent;
     height: 5px;
 }
 QScrollBar::handle:horizontal {
     background: #cccccc;;
     min-width: 100px;
 }
 QScrollBar::handle:horizontal:hover {
     background: #aaaaaa;
     min-width: 100px;
 }
 QScrollBar::add-line:horizontal {
	border:none;
	background:transparent;
 }
 QScrollBar::sub-line:horizontal {
	border:none;
	background:transparent;
 }</string>
           </property>
           <property name="iconSize">
            <size>
             <width>24</width>
             <height>24</height>
            </size>
           </property>
           <property name="verticalScrollMode">
            <enum>QAbstractItemView::ScrollPerPixel</enum>
           </property>
           <column>
            <property name="text">
             <string>   文件名称</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>   修改时间</string>
            </property>
           </column>
           <column>
            <property name="text">
             <string>   备注</string>
            </property>
           </column>
          </widget>
         </item>
        </layout>
       </widget>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="maindlg.qrc"/>
 </resources>
 <connections/>
</ui>
